alerter/src/main/java/org/apache/hertzbeat/alert/reduce/AlarmConvergeReduce.java

```diff
@@ -55,25 +55,33 @@ public AlarmConvergeReduce(AlertConvergeDao alertConvergeDao) {
     @SuppressWarnings("unchecked")
     public boolean filterConverge(Alert currentAlert) {
         // ignore monitor status auto recover notice
-        if (currentAlert.getTags() != null && currentAlert.getTags().containsKey(CommonConstants.IGNORE)) {
-            return true;
-        }
-        if (currentAlert.getStatus() == CommonConstants.ALERT_STATUS_CODE_RESTORED) {
+        if ((currentAlert.getTags() != null && currentAlert.getTags().containsKey(CommonConstants.IGNORE))
+                || currentAlert.getStatus() == CommonConstants.ALERT_STATUS_CODE_RESTORED) {
```

Comment:
```suggestion
        if (currentAlert.getStatus() == CommonConstants.ALERT_STATUS_CODE_RESTORED) {
```
status always is the ALERT_STATUS_CODE_RESTORED when tags has CommonConstants.IGNORE

Reply:
In both cases, the data in the map needs to be deleted，if remove （|| currentAlert . getStatus () == CommonConstants . ALERT_STATUS_CODE_RESTORED ），So when the label is not IGNORE，Unable to clear cache